gecode/kernel/core.hpp

```diff
@@ -1621,13 +1621,28 @@ namespace Gecode {
       /// Information is provided by a portfolio-based engine
       PORTFOLIO
     };
+    /// Reason for restarting 
+    enum RReason {
```

Comment:
We can probably afford expanding it to `RestartReason`.

Reply:
Sounds good. I've amended the code